app/src/main/java/com/example/memeapp/presentation/adapter/ItemClicked.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.memeapp.presentation.adapter
+
+interface ItemClicked {
```

Comment:
as you use this only in the adapter it can be on the adapter

Reply:
Done thx